### ISSUE TITLE
About generator of StarGAN-VC2

### ISSUE THREAD

**Issue by xieyuankun** [SamuelBroughton/StarGAN-Voice-Conversion-2#5]
I just run your code of StarGANv1 and v2. V1 works very well, But V2 only generate noise and the value of the discriminator is negative or NAN. I found that the generator of original paper used both original spk labels and target spk labels as input, but your implement only used target spk labels in generator which is same as the generator of StarGANv1. Is this the cause of the error？

**Comment by shackysureshot** [comment]
Hi, yes the current version of this repository does not include the source-and-target adv loss as reported in the paper. This will be pushed in the coming weeks, I'll leave this issue open until then.

**Comment by xieyuankun** [comment]
> Hi, yes the current version of this repository does not include the source-and-target adv loss as reported in the paper. This will be pushed in the coming weeks, I'll leave this issue open until then.

Okay, Thanks!

**Comment by tranctan** [comment]
Hi @xieyuankun, may I ask if the StarGANv1 generates reasonable results ? Since the author has not included sample results. It would be great if you can provide some generated results for reference :D. Thank you.

**Comment by shackysureshot** [comment]
@tranctan, I've just included some VCC2018 converted samples on the VC1 repository now, I'm yet to include information on running on VCC2018 dataset. Hope thats okay in the meantime

**Comment by tranctan** [comment]
Hi @SamuelBroughton, thank you so much for the work and quick reply. I just checked them out, they're basically well converted, but not as intelligible as the paper's demo. May I ask if this can be improved ( like tuning hyperparams, etc.) or it depends on the dataset ? Thank you so much.

**Comment by shackysureshot** [comment]
Thanks for the support @tranctan, yes, tweaking hyperparams etc. you may be able to find a better output. I am in the process of conducting subjective listening experiments using the current output of that model and am happy to share my results on completion of my report.

**Comment by tranctan** [comment]
I am looking forward to your results report. Any details such as number of iterations, estimated time of training, how to tune this model or interpret the loss logs would help a lot !!

**Comment by dipjyoti92** [comment]
Hi!

You may check the following repository that includes source-and-target adv loss as reported in the paper. 

https://github.com/dipjyoti92/StarGAN-Voice-Conversion-2

**Comment by xieyuankun** [comment]
> Hi!
> 
> You may check the following repository that includes source-and-target adv loss as reported in the paper.
> 
> https://github.com/dipjyoti92/StarGAN-Voice-Conversion-2

Thanks for your repository, I will try it！

**Comment by shackysureshot** [comment]
Commit https://github.com/SamuelBroughton/StarGAN-Voice-Conversion-2/commit/26484a2db5a00247535c7ec7c1f97c10e711e022
